### PR TITLE
Feature/topic 182

### PR DESCRIPTION
**Pull request by MarcoInnovations**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.3.1
\ No newline at end of file
+0.3.2
```
Comment (radepal) [inline VERSION]:
```suggestion
0.4.0
```

**Comment by radepal** [comment]
Missing CHANGELOG.md update

File: VERSION (hunk 1 above)
Comment (MarcoInnovations) [reply VERSION]: Why do you want to go to 0.4.0?  Is there a specific reasoning?

Comment (radepal) [reply VERSION]: Because terraform-aws-eks is uptaded to use new k8s higer version
is bigger update

**Pull request merged**